src/js/components/Box.js

```diff
@@ -3,11 +3,16 @@
 import React, { Component, PropTypes } from 'react';
 import KeyboardAccelerators from '../utils/KeyboardAccelerators';
 import Intl from '../utils/Intl';
+import Props from '../utils/Props';
 import SkipLinkAnchor from './SkipLinkAnchor';
 
 const CLASS_ROOT = "box";
 
 export default class Box extends Component {
+  constructor(props) {
+    super(props);
+    this.restProps = Props.omit(props, Object.keys(Box.propTypes));
```

Comment:
This doesn't handle the case where the caller changes property values after the initial render. It would need similar code in componentWillReceiveProps.